linters/namedfields/namedfields.go

```diff
@@ -0,0 +1,107 @@
+package namedfields
```

Comment:
nitpicking: for clarity I would add at least `init` suffix to the naming in general (package, structs, etc); this is because for me `namedfieldsinit`-linter sounds way more precise than just `namedfields`-linter (this can make more sense if we are planning to add more and more linters over time)